roles/repository/defaults/main.yml

```diff
@@ -1,3 +1,2 @@
 ---
-repository_key_files: []
-repository_key_files_dir: "/opt/configuration/environments/generic/repokeys"
+repository_key_files_directory: "/opt/configuration/environments/generic/repokeys"
```

Comment:
As mentioned in my first review please use ``files/repository/keys`` here.

Reply:
where do you mention this? I cannot find.
do you mean /opt/configuration/environments/generic/files/repository/keys ?